scripts/west_commands/runners/jlink.py

```diff
@@ -69,6 +70,11 @@ def do_add_parser(cls, parser):
                             help='J-Link Commander, default is JLinkExe')
         parser.add_argument('--erase', default=False, action='store_true',
                             help='if given, mass erase flash before loading')
+        parser.add_argument('--jlinkscriptfile', default='',
+                            help='J-Linkscriptfiles are mainly used to connect '
```

Comment:
Maybe just "path to J-Link script file"?

Reply:
ok